build_tools/format-diff.sh

```diff
@@ -54,15 +54,17 @@ fi
 set -e
 
 uncommitted_code=`git diff HEAD`
-LAST_MASTER=`git merge-base master HEAD`
+[ "$FORMAT_UPSTREAM" ] || FORMAT_UPSTREAM=origin/master
```

Comment:
Will it still work on private fork where master isn't sync to upstream master?

Reply:
Instead of 'origin' we could use `git remote -v | grep 'facebook/rocksdb.git' | head -n 1 | cut -f 1`